web/modules/custom/do_base/tests/src/Functional/PreviewLinkTest.php

```
@@ -0,0 +1,231 @@
+    $node = $this->createModeratedNode('[TEST] Expired Draft', 'draft');
+    $preview_link = $this->createPreviewLink($node);
+    $url = $preview_link->getUrl($node);
+
+    // Act.
+    $this->drupalGet($url);
+
+    // Assert.
+    $this->assertSession()->statusCodeEquals(200);
+
+    // Prepare.
+    $preview_link->setExpiry(new \DateTime('-1 minute'));
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '121', column '34'). (undefined)

(MissingImport)